config/client_config/me/system/config/templates/features/aca_individual_market/financial_assistance.yml

```diff
@@ -90,7 +90,7 @@ registry:
       - key: :immigration_status_question_required
         is_enabled: false
       - key: :mec_check
-        is_enabled: true
+        is_enabled: <%= ENV['MEC_CHECK_IS_ENABLED'] || false %>
```

Comment:
We should NOT be setting the default to false here. Since this feature is expected to be turned on in prod.